site/content/guides/external-authorization.md

```diff
@@ -361,13 +361,171 @@ $ curl -k --user user1:password1 https://local.projectcontour.io/test/$((RANDOM)
 {"TestId":"","Path":"/test/27132","Host":"local.projectcontour.io","Method":"GET","Proto":"HTTP/1.1","Headers":{"Accept":["*/*"],"Auth-Handler":["htpasswd"],"Auth-Realm":["default"],"Auth-Username":["user1"],"Authorization":["Basic dXNlcjE6cGFzc3dvcmQx"],"Content-Length":["0"],"User-Agent":["curl/7.64.1"],"X-Envoy-Expected-Rq-Timeout-Ms":["15000"],"X-Envoy-Internal":["true"],"X-Forwarded-For":["172.18.0.1"],"X-Forwarded-Proto":["https"],"X-Request-Id":["2c0ae102-4cf6-400e-a38f-5f0b844364cc"],"X-Request-Start":["t=1601601826.102"]}}
 ```
 
+## Global External Authorization
```

Comment:
Please move this content into `site/content/docs/main/guides/external-authorization.md` instead, we're now versioning the guides along with the rest of the docs. It will be copied from there into the 1.25 directory when we cut a release.